### PR TITLE
CAPT-218 Remove unnecessary encryption checks 

### PR DESCRIPTION
**Pull request by todor-kolev**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,7 +27,7 @@ DOCS_VERIFY_SKIP ?= false
 
 DOCKER_BUILD_ARGS := $(if $(DOCKER_VERSION), "--build-arg=DOCKER_VERSION=$(DOCKER_VERSION)",)
 DOCKER_RUN_OPTS := $(TRAEFIK_ENVS) $(TRAEFIK_MOUNT) "$(TRAEFIK_DEV_IMAGE)"
-DOCKER_RUN_TRAEFIK := docker run $(INTEGRATION_OPTS) -i $(DOCKER_RUN_OPTS)
+DOCKER_RUN_TRAEFIK := docker run $(INTEGRATION_OPTS) -it $(DOCKER_RUN_OPTS)
```
Comment (hughfdjackson) [inline Makefile]: While this gets us up and running, it also may break several other pieces of functionality in this make file :thinking:

Comment (hughfdjackson) [reply Makefile]: I think we should take a more considered look at this.

Comment (hughfdjackson) [reply Makefile]: If we remove this for now, I'd happily merge this PR.

**Review by hughfdjackson — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -27,7 +27,7 @@ DOCS_VERIFY_SKIP ?= false
...
 DOCKER_RUN_TRAEFIK_NOTTY := docker run $(INTEGRATION_OPTS) -i $(DOCKER_RUN_OPTS)
```
Comment (hughfdjackson) [inline Makefile]: Didn't mean to approve!

**Review by hughfdjackson — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (todor-kolev) [reply Makefile]: Removed

**Review by hughfdjackson — APPROVED** [review]

**Pull request merged**